chromium_src/chrome/browser/ui/tabs/public/tab_features.h

```diff
@@ -12,7 +12,7 @@
 
 #define Init(...)                 \
```

Comment:
is this override still needed if Init is now virtual?

Reply:
We don't currently subclass `TabFeatures`, so I think it is still needed at the moment.